### ISSUE TITLE
Have local version and released version of the plugin running

### ISSUE THREAD

**Issue by KevinGutowski** [skpm/skpm#204]
I realize this is probably a bit complicated to support but it would be nice to have some way to run both the released version of the plugin and the local version of the plugin that I am producing. Currently when I attempt to install the released version I have to override my local instance.

**Comment by mathieudutour** [comment]
Good point. I don’t think I’d be too hard actually: we can append `.dev` to the plugin identifier when `process.env.NODE_ENV !== 'production'`. It is set to production when publishing so it would work fine I think
